docs/demo/gaiden-css/components/list/with_icons/demo.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+
+  <style>
+    body {
+      background-color: #f9f9f9;
+    }
+  </style>
+</head>
+
+<body>
+  <article>
+    <header>
+      <h5 class="list__title">Topic List Header</h5>
+    </header>
+    <nav>
+      <ul class="list list--with-icons">
+        <li class="list__item" role="none">
```

Comment:
`role=“none”` removes the semantic value of this element. We need to remove this on our sample and website

Reply:
Removed! :wink